### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by eliascarv**
Update tests so they work on CI and different versions of Julia.

### CODE REVIEW

**Comment by eliascarv** [comment]
This PR is ready for review @juliohm.

File: test/transforms.jl [hunk 1]
```diff
@@ -214,7 +216,8 @@
       p₂ = scatter(n.x, n.y, label="Center")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,  "center.png") p
+      @test_reference joinpath(datadir, "center.png") p
+      # savefig(p, joinpath(datadir,  "center.png"))
```
Comment (juliohm) [inline test/transforms.jl:220]:
```suggestion
```

File: test/transforms.jl [hunk 2]
```diff
@@ -228,18 +231,18 @@
...
     T = Scale(low=0, high=1)
     n, c = apply(T, t)
-    @test all(x -> x <= 1, n.x)
-    @test all(x -> x >= 0, n.x)
-    @test all(y -> y <= 1, n.y)
-    @test all(y -> y >= 0, n.y)
+    @test all(≤(1), n.x)
+    @test all(≥(0), n.x)
+    @test all(≤(1), n.y)
+    @test all(≥(0), n.y)
```
Comment (juliohm) [inline test/transforms.jl:245]: Awesome style!

File: test/transforms.jl [hunk 3]
```diff
@@ -249,14 +252,15 @@
       p₂ = scatter(n.x, n.y, label="Scale")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,"scale.png") p
+      @test_reference joinpath(datadir, "scale.png") p
+      # savefig(p, joinpath(datadir, "scale.png"))
```
Comment (juliohm) [inline test/transforms.jl:256]:
```suggestion
```

File: test/transforms.jl [hunk 4]
```diff
@@ -275,7 +279,8 @@
       p₂ = scatter(n.x, n.y, label="ZScore")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,"zscore.png") p
+      @test_reference joinpath(datadir, "zscore.png") p
+      # savefig(p, joinpath(datadir, "zscore.png"))
```
Comment (juliohm) [inline test/transforms.jl:283]:
```suggestion
```

File: test/transforms.jl [hunk 5]
```diff
@@ -304,8 +309,8 @@
     t = Table(; x, y)
     T = Functional(cos)
     n, c = apply(T, t)
-    @test all(x -> -1 <= x <= 1, n.x)
-    @test all(y -> -1 <= y <= 1, n.y)
+    @test all(x -> -1 ≤ x ≤ 1, n.x)
+    @test all(y -> -1 ≤ y ≤ 1, n.y)
```
Comment (juliohm) [inline test/transforms.jl:313]: 💯

File: test/transforms.jl [hunk 6]
```diff
@@ -424,24 +429,26 @@
       p = plot(p₁, p₂, p₃, p₄, layout=(2,2))
       q = plot(p₂, p₃, p₄, p₅, p₆, p₇, layout=(2,3))
 
-      @test_reference joinpath(datadir,"eigenanalysis-1.png") p
-      @test_reference joinpath(datadir,"eigenanalysis-2.png") q
+      @test_reference joinpath(datadir, "eigenanalysis-1.png") p
+      @test_reference joinpath(datadir, "eigenanalysis-2.png") q
+      # savefig(p, joinpath(datadir, "eigenanalysis-1.png"))
+      # savefig(q, joinpath(datadir, "eigenanalysis-2.png"))
```
Comment (juliohm) [inline test/transforms.jl:435]:
```suggestion
```

**Review by juliohm — CHANGES_REQUESTED** [review]
Amazing @eliascarv , just a few minor suggestions.

File: test/transforms.jl (hunk 6 above)
Comment (eliascarv) [reply test/transforms.jl]: I removed the comments of savefig function.

File: test/transforms.jl (hunk 2 above)
Comment (eliascarv) [reply test/transforms.jl:244]: Thanks, `≤(1)` it's equivalent to `x -> x ≤ 1`

File: test/transforms.jl (hunk 5 above)
Comment (eliascarv) [reply test/transforms.jl:310]: Thanks! I love unicode!

File: test/transforms.jl (hunk 4 above)
Comment (eliascarv) [reply test/transforms.jl]: I removed the comments of savefig function.

File: test/transforms.jl (hunk 3 above)
Comment (eliascarv) [reply test/transforms.jl]: I removed the comments of savefig function.

**Comment by juliohm** [comment]
Fixes #18

File: test/transforms.jl (hunk 1 above)
Comment (eliascarv) [reply test/transforms.jl]: I removed the comments of savefig function.

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
Awesome work @eliascarv. Feel free to take a new issue from the list of open issues 👍🏽

**Pull request merged**